### PR TITLE
RELATED: BB-1110 Add AM typings

### PR DESCRIPTION
**Pull request by petrjanu**

### CODE REVIEW

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1638/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1107/ : SUCCESS in 8m 17s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/645/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1639/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/350/ : SUCCESS in 15m 18s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/359/ : SUCCESS in 27s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1109/ : SUCCESS in 14m 06s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/646/ : SUCCESS in 13m 39s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/275/ : FAILURE in 19m 04s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1645/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/353/ : FAILURE in 10m 43s

**Comment by petrjanu** [comment]
Cannot pass examples test since there is some issue with Examples project data. Fix data and merge again.

**Comment by martinmosko** [comment]
LGTM

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/360/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1112/ : SUCCESS in 2m 29s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/649/ : UNSTABLE in 11m 19s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/278/ : FAILURE in 8m 57s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1652/ : SUCCESS in 27s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1117/ : SUCCESS in 9m 10s

File: examples/test/Visualization_test.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ test('Chart visualization should render', async (t) => {
         .expect(chart.exists)
         .ok()
         .expect(chart.textContent)
-        .eql('Created with Highcharts 4.2.5Month/Year (Date)$ Total Sales$2,621,884$2,625,589$984,801Oct 2017Nov 2017Dec 20170M1M2M3M');
```
Comment (tomassatinsky) [inline examples/test/Visualization_test.js]: What caused this change?

Comment (petrjanu) [reply examples/test/Visualization_test.js]: Tested visualization has now static period filter.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/363/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1120/ : SUCCESS in 2m 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/653/ : SUCCESS in 12m 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/281/ : SUCCESS in 20m 30s

**Pull request merged**